### PR TITLE
docs: say that a page is a text file and link to editors

### PR DESCRIPTION
**Pull request by italicize**
A few extra details to help noobs.

### CODE REVIEW

File: docs/tutorial-create-pages.md [hunk 1]
```diff
@@ -33,8 +33,10 @@ function HelloWorld(props) {
 module.exports = HelloWorld;
 ```
 
+> Use any text editor to make the file, such as [Microsoft Visual Studio Code](https://code.visualstudio.com/download) or [Komodo Edit](https://www.activestate.com/komodo-edit).
+
 2. Go to http://localhost:3000/hello-world and you should be able to see the new page.
-1. Change the text within the `<p>...</p>` to "I can write JSX here!". The browser should refresh automatically to reflect the changes.
+1. Change the text within the `<p>...</p>` to "I can write JSX here!" and save the file again. The browser should refresh automatically to reflect the change.
```
Comment (wgao19) [inline docs/tutorial-create-pages.md]: The list number is off.

**Comment by wgao19** [comment]
Again need to update in all the versioned docs.

File: docs/tutorial-create-pages.md (hunk 1 above)
Comment (yangshun) [reply docs/tutorial-create-pages.md]: Markdown syntax will continue from the previous number, so it's fine (click the preview to have a look).

**Comment by yangshun** [comment]
> Again need to update in all the versioned docs.

It's ok, we always point to the `next` (latest) version anyway.

**Pull request merged**